deny.toml

```diff
@@ -44,8 +44,8 @@ skip-tree = [
     # wait for curve25519-dalek to upgrade
     { name = "digest", version = "=0.9.0" },
 
-    # wait for primitive-types to upgrade
-    { name = "proc-macro-crate", version = "=0.1.5" },
+    # wait for jsonrpc-http-server to upgrade tokio-util, parking_lot
```

Comment:
I tweaked this a bit to list the dependencies that are actually causing the problems here. It's tricky to work that out because the dependency trees are complicated.

I also removed some exceptions that other upgrades had made obsolete.